project.clj

```diff
@@ -1,10 +1,10 @@
-(defproject onaio/optimus-sass "0.1.0"
+(defproject optimus-sass "0.2.0"
```

Comment:
why change the project name? is this to align it w/patterns in our other libraries?

Reply:
seems like not, https://github.com/onaio/hatti/blob/master/project.clj#L6

Reply:
reverted this change